router/core/router_config.go

```diff
@@ -149,6 +149,7 @@ func (c *Config) Usage() map[string]any {
 	usage["apollo_compatibility_flags_suppress_fetch_errors_enabled"] = c.apolloCompatibilityFlags.SuppressFetchErrors.Enabled
 	usage["apollo_compatibility_flags_truncate_floats_enabled"] = c.apolloCompatibilityFlags.TruncateFloats.Enabled
 	usage["apollo_compatibility_flags_value_completion_enabled"] = c.apolloCompatibilityFlags.ValueCompletion.Enabled
+	usage["apollo_compatibility_flags_use_graphql_validation_failed_status_enabled"] = c.apolloCompatibilityFlags.UseGraphQLValidationFailedStatus.Enabled
```

Comment:
Above this, `usage["apollo_compatibility_flags_replace_undefined_op_field_errors_enabled"] = c.apolloCompatibilityFlags.ReplaceUndefinedOpFieldErrors.Enabled`